src/components/BlockOverview.vue

```diff
@@ -58,15 +90,23 @@
 
 <script>
 const { formatXe } = require('@edge/wallet-utils')
+import { ClipboardCopyIcon } from '@heroicons/vue/outline'
 
 export default {
   name: "BlockOverview",
+  components: {
+    ClipboardCopyIcon
+  },
   props: {
     block: {
       type: Object
     }
   },
   methods: {
+    copyToClipboard(input) {
+      if (!!navigator.clipboard) window.alert('Clipboard unavailable. Please copy-paste manually.')
+      return navigator.clipboard.writeText(input)
```

Comment:
`writeText` will be executed even if `navigator.clipboard` resolves to false. Please ensure in all cases that this is only called if the clipboard is available.